### DISCUSSION TITLE
quasar-admin is not running.

### DISCUSSION THREAD

**Discussion by topdevhere623** [pratik227/quasar-admin#38] [General]
Hello, @pratik227!

Thank you for making amazing Quasar Project with Vue3.
I've investigated the admin panel in quasar-admin and wanted to clone and run on my local.

But I have faced some issues on running the project.

![image](https://user-images.githubusercontent.com/93789798/234105820-8b7b540c-b358-40fe-8424-d5f565c7979c.png)
![image](https://user-images.githubusercontent.com/93789798/234105879-9c1163a5-8eac-4ed6-af25-a1d29a56311d.png)

I have attached some issues, hope you will help me.

thank you.

**Comment by pratik227** [comment]
Hi,

I think your local quasar cli is old. Try to uninstall it and install it
again.

https://quasar.dev/start/quasar-cli

Thanks,
Pratik Patel

**Comment by topdevhere623** [comment]
Thank you, it worked.
